docs/DeveloperGuide.adoc

```diff
@@ -721,6 +721,30 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
+|`* * *` |busy user |search using accumulative filters |narrow down my search in the shortest time possible
+
+|`* * *` |careless user that cant remember the order of entry |add a contact without following the particular order of entry |not remember the order of entry
```

Comment:
"careless user that find it troublesome to follow a particular order of entering contact information"
"add contact information without following any particular order"
"enter contact information more easily and conveniently"